src/platform/datapath_raw.c

```diff
@@ -117,6 +117,17 @@ CxPlatDataPathInitialize(
 
     UNREFERENCED_PARAMETER(TcpCallbacks);
 
+    if (NewDataPath == NULL) {
+        Status = QUIC_STATUS_INVALID_PARAMETER;
+        goto Exit;
```

Comment:
It's fine to use `return` instead.

Reply:
This style is following the style of the other implementations of CxPlatDataPathInitialize in winuser, winkernel, etc. I'd rather not make this style change in this PR.